output.log

```diff
@@ -0,0 +1,2 @@
+Starting audit...
+Audit done.
```

Comment:
What is this file?

Reply:
ci item is fixed to create this file in `.ci-temp`

Reply:
CI failure
> ./.ci/travis/travis.sh: line 174: .ci-temp/output.log: No such file or directory

Reply:
fixed.